src/components/About/About.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Image from 'next/image'
+
+export const About = () => {
+  return (
+    <div>
+      <section id="about-us">
+        <div class="container-fluid">
```

Comment:
```suggestion
        <div className="container-fluid">
```

Replace all classs with ClassName to fix this working 
![image](https://user-images.githubusercontent.com/64401853/220031186-37967dfd-41f5-4af1-b1b0-457aca6f0b22.png)